Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp

```diff
@@ -87,21 +88,26 @@ ErrorOr<void> VideoPlayerWidget::setup_interface()
     find_descendant_of_type_named<GUI::Button>("fullscreen")->set_action(*m_toggle_fullscreen_action);
 
     m_size_fit_action = GUI::Action::create_checkable("&Fit", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Fit);
+        set_sizing_mode(VideoSizingMode::Fit);
     });
 
     m_size_fill_action = GUI::Action::create_checkable("Fi&ll", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Fill);
+        set_sizing_mode(VideoSizingMode::Fill);
     });
 
     m_size_stretch_action = GUI::Action::create_checkable("&Stretch", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::Stretch);
+        set_sizing_mode(VideoSizingMode::Stretch);
     });
 
     m_size_fullsize_action = GUI::Action::create_checkable("F&ull Size", [&](auto&) {
-        m_video_display->set_sizing_mode(VideoSizingMode::FullSize);
+        set_sizing_mode(VideoSizingMode::FullSize);
     });
 
+    // Load the current video sizing mode
+    // The default fallback for `read_u32` is 0, which is also our desired defualt for the sizing mode, VideoSizingMode::Fit
+    auto sizing_mode_value = Config::read_u32("VideoPlayer"sv, "Playback"sv, "SizingMode"sv);
+    set_sizing_mode(static_cast<VideoSizingMode>(sizing_mode_value));
```

Comment:
If we get an invalid (out-of-range) integer value here, casting it to a `VideoSizingMode` leads to undefined behavior.